src/styles/Editor.scss

```diff
@@ -24,7 +24,23 @@
   }
 }
 
+.program-sketch-thumbnail {
+  height: 50px;
+  width: 50px;
+  padding: 4px;
+  border-radius: 25px;
+
+  color: $theme-dark;
+  @include themify($themes) {
+    color: themed("cardColor");
+    background-color: themed("cardBackground")
+}
```

Comment:
I'm a bit confused, what's the point of setting the `color`? We're not putting any text in this thumbnail :O